addons/sys_core/fnc_setSpokenLanguages.sqf

```diff
@@ -16,11 +16,8 @@
 #include "script_component.hpp"
 
 private _list = _this;
-private _languageIds = [];
+private _languageIds = _list apply {([_x] call FUNC(getLanguageId))};
```

Comment:
What do you have with parenthesis! :P
`private _languageIds = _list apply {[_x] call FUNC(getLanguageId)};`